content/docs/tutorials/getting-started-aws-letsencrypt/README.md

```diff
@@ -0,0 +1,654 @@
+> according to the Amazon's [EKS Best Practices Guide](https://aws.github.io/aws-eks-best-practices/).
+> Consider using the [AWS Load Balancer Controller](https://docs.aws.amazon.com/eks/latest/userguide/aws-load-balancer-controller.html) instead.
+
+The stable DNS host name of the load balancer can be used as an alias for the `www` record in your chosen `$DOMAIN_NAME`
+by creating a Route53 [Alias Record](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/resource-record-sets-choosing-alias-non-alias.html):
+
+```bash
+HOSTED_ZONE_ID=$(aws route53 list-hosted-zones-by-name --dns-name $DOMAIN_NAME --query "HostedZones[0].Id" --output text)
+ELB_CANONICAL_HOSTED_ZONE_NAME=$(kubectl get svc helloweb --output=jsonpath='{ .status.loadBalancer.ingress[0].hostname }')
+aws elb describe-load-balancers --query "LoadBalancerDescriptions[?CanonicalHostedZoneName == '$ELB_CANONICAL_HOSTED_ZONE_NAME'] | [0]" \
+| jq '{
+  "Comment": "Creating a CNAME record",
+  "Changes": [
+    {
+      "Action": "CREATE",
+      "ResourceRecordSet": {
+        "Name": "www.\($DOMAIN_NAME)",
+        "Type": "A",
+        "AliasTarget": {
+          "HostedZoneId": .CanonicalHostedZoneNameID,
+          "DNSName": .CanonicalHostedZoneName,
+          "EvaluateTargetHealth": false
+        }
+      }
+    }
+  ]
+}' \
+    --arg DOMAIN_NAME "${DOMAIN_NAME}" \
+| aws route53 change-resource-record-sets --hosted-zone-id $HOSTED_ZONE_ID --change-batch file:///dev/stdin
+```
+
+> ℹ️ Read [Routing traffic to an ELB load balancer](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/routing-to-elb-load-balancer.html)
+> to learn more about this task.
+>
+> ℹ️ The script uses a [`JMESPath` query](https://docs.aws.amazon.com/cli/latest/userguide/cli-usage-filter.html)
+> to get the ELB for the Kubernetes Service by matching against the DNS name.
+
+Check that `www.$DOMAIN_NAME` now resolves to the ephemeral public IP address of the load balancer:
+
+```terminal
+$ dig www.$DOMAIN_NAME A
+...
+;; QUESTION SECTION:
+;www.cert-manager-aws-tutorial.richard-gcp.jetstacker.net. IN A
+
+;; ANSWER SECTION:
+www.cert-manager-aws-tutorial.richard-gcp.jetstacker.net. 60 IN A 34.212.236.229
+www.cert-manager-aws-tutorial.richard-gcp.jetstacker.net. 60 IN A 44.232.234.71
+www.cert-manager-aws-tutorial.richard-gcp.jetstacker.net. 60 IN A 35.164.69.198
+```
+
+If the DNS is correct and the load balancer is working and the hello world web server is running,
+you should now be able to connect to it using curl or using your web browser:
+
+```bash
+curl --insecure -v https://www.$DOMAIN_NAME
+```
+
+> ⚠️ We used curl's `--insecure` option because it rejects self-signed certificates by default.
+> Later you will learn how to create a trusted certificate signed by Let's Encrypt.
```

Comment:
suggestion: This doesn't quite read right (it sounds like it's the `insecure` flag which rejects self signed certs).

```suggestion
> ⚠️ We used curl's `--insecure` option because curl will reject the untrusted certificate we generated otherwise.
> Later you will learn how to create a trusted certificate signed by Let's Encrypt.
```